pytest.ini

```diff
@@ -1,3 +1,6 @@
 [pytest]
 DJANGO_SETTINGS_MODULE=pycon.dev-settings
-addopts = --nomigrations --cov-config .coveragerc --cov ./ --cov-report term -n auto
+addopts = --nomigrations --cov-config .coveragerc --cov ./ --cov-report term
```

Comment:
How come you're removing the multi-threading from the tests?

Reply:
good point – I now fixed this in Makefile.

The gist of it is – the -n auto is not super smart and it just creates N runners where N is number of CPU cores available (including hyperthreading cores) – so for example on my laptop it wants to create 4 workers, with 4 databases. If I want to run a single file, it takes *much* more time to set up the test than to run the test, and significant part of that is due to addtional 3 runners.

However – there is some gain on the total speed when running the complete testsuite, so I just moved the -n auto from pytest.ini to Makefile:test.

I also thought about putting a static number there – say `-n 2` but that doesn't seem to solve the original problem – however – it is faster (on my slow laptop) than the 4 workers, when you include the total setup time.